app/models/pacbio/run.rb

```diff
@@ -53,7 +53,13 @@ def comments
     # using pipelines.yml configuration to generate data
     def generate_sample_sheet
       configuration = pacbio_run_sample_sheet_config
-      sample_sheet = RunCsv::PacbioSampleSheet.new(object: self, configuration:)
+      sample_sheet_class =
+        if Flipper.enabled?(:new_format_sample_sheet) && (smrt_link_version.name == 'v13_revio')
```

Comment:
Probably a naive question but what happens once the feature flag is removed?

Reply:
OK. Glad I asked the question. Would it be better to move this to configuration after the feature flag is gone? I don't suppose the original PacbioSampleSheet will be going any time soon as long as we are using the Sequel IIe. This also feels like a good implementation for being able to have different sample sheet classes.